boilerplate/openshift/golang-osd-operator/standard.mk

```
@@ -108,9 +108,16 @@ coverage:
 	boilerplate/openshift/golang-osd-operator/codecov.sh
 
 .PHONY: test
-test: gotest yaml-validate
+test: gotest validate-olm-deploy-yaml
 
-.PHONY: yaml-validate
-yaml-validate:
+.PHONY: install-pyyaml
+install-pyyaml:
 	python3 -mpip install pyyaml
+
+.PHONY: validate-olm-deploy-yaml
+validate-olm-deploy-yaml: install-pyyaml
+	python3 boilerplate/openshift/golang-osd-operator/validate-yaml.py $(shell git ls-files | egrep '^(deploy)/' | egrep '.*\.ya?ml')
```

Since we're being inclusive rather than exclusive, `ls-files` lets you use shell globs.

The quotes are necessary to avoid premature shell expansion.

The wildcarding *does* recurse into subdirectories, which is what we want.

```suggestion
	python3 boilerplate/openshift/golang-osd-operator/validate-yaml.py $(shell git ls-files 'deploy/*.yaml' 'deploy/*.yml')
```

---

Couldn't you then `$(shell git ls-files | egrep '^deploy.*\.ya?ml$')` ?